src/sentry/constants.py

```diff
@@ -334,3 +340,15 @@ def as_choices(cls):
 
 
 StatsPeriod = namedtuple('StatsPeriod', ('segments', 'interval'))
+
+FILTER_STAT_KEYS_TO_VALUES = BidirectionalMapping({
```

Comment:
I don't remember what I was on about here before. A dictionary should be fine.

Reply:
I'm a little worried that since these are defined so far away from where they are used in `should_filter` that they won't updated as fields are added/removed/changed:

https://github.com/getsentry/sentry/blob/a4862a028efee45baaaf975bcba181bcb737a83d/src/sentry/coreapi.py#L389-L414

Reply:
It's not actually used in `should_filter`, either - it's used in `web/api.py` when `should_filter()[0]` returns True and it's used in `project_stats.py`. Would it be better to declare in `web/api.py`?

Reply:
I just meant the stat key definition (e.g. `ip-address`.) It looks like this is updated as of the latest revision (0dad70c1d88cc1e22e7a7dfd51caae300564da6a) and is much better. 👍